### PR TITLE
feat(clever): implement farms and abcCVX

### PR DESCRIPTION
**Pull request by 4E-69-63-6B-20-42-6F-73**
## Description

Implemented farms for Clever and abcCVX.

## Checklist

- [X] I have followed the [Contributing Guidelines](../CONTRIBUTING.md)
- [X] (optional) As a contributor, my Ethereum address/ENS is: 0xCfb3B99e0A7Ad8008A5Bd5D3255e338f493F8c15
- [ ] (optional) As a contributor, my Twitter handle is:

## How to test?

The address 0xCfb3B99e0A7Ad8008A5Bd5D3255e338f493F8c15 participates in the abcCVX farm.

### CODE REVIEW

**Comment by 4E-69-63-6B-20-42-6F-73** [comment]
I have no idea why the build failed. It has something to do with the generation of files.

File: src/apps/clever/contracts/abis/clever-gauge.json [hunk 1]
```diff
@@ -0,0 +1,562 @@
...
+    ],
+    "outputs": []
+  },
+  {
+    "stateMutability": "view",
+    "type": "function",
+    "name": "decimals",
+    "inputs": [],
+    "outputs": [{ "name": "", "type": "uint256" }],
+    "gas": 360
```
Comment (wpoulin) [inline src/apps/clever/contracts/abis/clever-gauge.json:80]: Remove all "gas" properties within the .json files, then delete /ethers folder + index and regenerate the contracts will resolve the issue

Comment (4E-69-63-6B-20-42-6F-73) [reply src/apps/clever/contracts/abis/clever-gauge.json]: Done!

**Review by wpoulin — APPROVED** [review]

**Comment by wpoulin** [comment]
Hey @4E-69-63-6B-20-42-6F-73 ,
I was testing out the farm position and found out something's wrong with the abc token fetcher. Price and liquidity are null and this leads to missing supplied balances to the farm position. While linting/rafactoring I also discovered abc tokens are coming from Cencentrator so it should in the concentrator app, not in the clever one.

I didn't want you to lose the work you've done, so I've commented out the ABC token fetcher until it's fixed

**Comment by 4E-69-63-6B-20-42-6F-73** [comment]
Hey @wpoulin

Oh noo!, will fix it!
Ps, is there a way to see how Zapper UI will render the response?

**Pull request merged**